src/main/java/com/bookstore/controller/AuthenticationController.java

```diff
@@ -16,7 +16,7 @@
 import jakarta.validation.Valid;
 
 @RestController
-@RequestMapping("/auth")
+@RequestMapping("/api/auth")
```

Comment:
add `server.servlet.context-path=/api` to `application.properties`
```suggestion
@RequestMapping("/auth")
```